vyper/parser/stmt.py

```diff
@@ -1020,11 +1020,13 @@ def get_target(self, target):
                 "Cannot modify storage inside %s: %s" % (
                     self.context.pp_constancy(),
                     target.annotation,
-                )
+                ),
+                self.stmt,
             )
         if not target.mutable:
             raise ConstancyViolationException(
-                "Cannot modify function argument: %s" % target.annotation
+                "Cannot modify function argument: %s" % target.annotation,
+                self.stmt,
```

Comment:
Will it just concat the two strings together, or do you need a second `%s`?

Reply:
`self.stmt` is the AST object that gets passed to the exception constructor. It's a separate expression from the formatted string above it.  Doing that allows to the exception to be formatted with a message that shows the location of the error in the Vyper source file.